app/controllers/display_task_controller.rb

```diff
@@ -1,12 +1,16 @@
 class DisplayTaskController < ApplicationController
   def index
     @all_tasks = Task.all
+    @all_people = Person.all
+          
     render :index
   end
 
   def links
     @id = params[:id]
     @task = Task.find(@id)
+    person = @task.person_id
+    @person = person ? Person.find(person) : nil
```

Comment:
Great use of the ternary! If you use `find_by` rather than `find` it will automatically return nil if it hasn't found the model object.